drivers/media/platform/raspberrypi/pisp_be/pisp_be_formats.h

```diff
@@ -504,11 +504,6 @@ static const struct pisp_be_format supported_formats[] = {
 		.colorspace_mask    = V4L2_COLORSPACE_MASK_RAW,
 		.colorspace_default = V4L2_COLORSPACE_RAW,
 	},
-	/* Opaque BE format for HW verification. */
-	{
-		.fourcc		    = V4L2_PIX_FMT_RPI_BE,
-		.align		    = 32,
-	},
```

Comment:
This format is also used in the non-HoG cases for our pisp HW verification tests.  @njhollinghurst perhaps we can reintroduce this patch in our downstream tree to keep the tests working?